src/Link.js

```diff
@@ -20,6 +20,7 @@ class LinkWithNavigation extends Component {
       routeKey,
       navigation,
       action,
+      ...restProps,
```

Comment:
What if we renamed this to `anchorProps`, (with no "..." spreading) so that the use is explicit. I want to make sure we can add functionality to this component in the future, without breaking people's apps. This way we don't have the same namespace as the component